### PR TITLE
Convert kubemark

### PR DESCRIPTION
**Pull request by krzyzacy**
#1230 #1830 #1810 

Okay `E2E_TEST=false` need to be translated to `--test=false`, which causes the timeout

cc @shyamjvs, and I'll dry-run against my own project

Fixes https://github.com/kubernetes/kubernetes/issues/41108

### CODE REVIEW

**Comment by krzyzacy** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-kubemark-5-gce-1.5/20170208-021912--f712e9a6a5d6b6a-111489

should be good now

**Comment by krzyzacy** [comment]
@wojtek-t add you here to double check, I ran gce-5 in my own project and it passes now, I can shoot up a gce-500 locally tomorrow as well before I merge again. It should not change the job behavior though.

I'll change the cluster name https://github.com/kubernetes/kubernetes/issues/41108#issuecomment-278270042.

File: jobs/ci-kubernetes-kubemark-100-gce.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### job-env
+ENABLE_GARBAGE_COLLECTOR=true
+E2E_NAME=kubemark-100
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-kubemark-100-gce.env]: TODO!!! - move this to config.json and set in --cluster flag. I definitely wrote this flag and let it slipped :-| (too sleepy to update the PR now, so go dream..

File: jobs/config.json [hunk 2]
```diff
@@ -1337,6 +1337,65 @@
...
+    "--test=false"
+  ]
+},
+
+"ci-kubernetes-kubemark-gce-scale": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-kubemark-gce-scale.env",
+    "--docker-in-docker"
```
Comment (wojtek-t) [inline jobs/config.json]: --test=false

?

**Comment by wojtek-t** [comment]
I would like to go over those changes once this is ready for review.

File: jobs/config.json (hunk 2 above)
Comment (krzyzacy) [inline jobs/config.json]: done.

File: jobs/ci-kubernetes-kubemark-100-gce.env (hunk 1 above)
Comment (krzyzacy) [inline jobs/ci-kubernetes-kubemark-100-gce.env]: done.

**Comment by krzyzacy** [comment]
It's annoying github does not want to take it as renaming even I do `git mv`.

I would say let https://github.com/kubernetes/test-infra/pull/1836 in and then this one.

@wojtek-t PTAL

**Comment by wojtek-t** [comment]
LGTM - feel free to merge after the other PR.

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

**Comment by krzyzacy** [comment]
all green and jenkins is revived, merge to give it another shot

**Pull request merged**